src/MarginTrading.Backend.Contracts/IAccountHistoryApi.cs

```diff
@@ -1,21 +1,21 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using MarginTrading.Contract.BackendContracts;
+using MarginTrading.Backend.Contracts.AccountHistory;
 using Refit;
 
 namespace MarginTrading.Backend.Contracts
 {
     public interface IAccountHistoryApi
     {
         [Get("/api/accountHistory/byTypes")]
-        Task<AccountHistoryBackendResponse> ByTypes([Query] AccountHistoryBackendRequest request);
+        Task<AccountHistoryResponse> ByTypes([Query] AccountHistoryRequest request);
```

Comment:
Memo: maybe it's better to explicitly mark model as "...DataReaderResponse".

Reply:
I don't think so. DataReader is just a read part of backend.. Not now at least)